<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>622</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxRunningTextSetup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>740</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Futó szöveg beállításai</string>
    </property>
    <widget class="QLabel" name="labelSpeed">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sebesség</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollBarSpaceWidth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>81</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="pageStep">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="textSize">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontChanger">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>188</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveSetup">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>76</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Beállítások mentése</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpaceWidth">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Szóköz szélesség</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFont">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Betűtípus</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTextSize">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Méret</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runText">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>75</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <stylestrategy>NoAntialias</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>⏵</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollBarSpeed">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>81</y>
       <width>116</width>
       <height>18</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="pageStep">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewRunningTextSetup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>740</width>
       <height>130</height>
      </rect>
     </property>
    </widget>
    <zorder>graphicsViewRunningTextSetup</zorder>
    <zorder>labelSpeed</zorder>
    <zorder>scrollBarSpaceWidth</zorder>
    <zorder>textSize</zorder>
    <zorder>fontChanger</zorder>
    <zorder>saveSetup</zorder>
    <zorder>labelSpaceWidth</zorder>
    <zorder>labelFont</zorder>
    <zorder>labelTextSize</zorder>
    <zorder>runText</zorder>
    <zorder>scrollBarSpeed</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBoxImportText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>350</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Szöveg bevitele</string>
    </property>
    <widget class="QGraphicsView" name="graphicsViewImportText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>350</width>
       <height>250</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditEditText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>330</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Futó szöveg...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Bevitel</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewImportFromFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Fájl kiválasztása</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelChosenFileName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>227</y>
       <width>180</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nincs fájl kiválasztva.</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelReadFromFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>100</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Beolvasás fájlból</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecEyeTracking">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>350</width>
      <height>270</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Szemmozgás felvétele</string>
    </property>
    <widget class="QGraphicsView" name="graphicsViewRecEyeTracking">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>350</width>
       <height>140</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="runCalibration">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Kalibráció indítása</string>
     </property>
    </widget>
    <widget class="QPushButton" name="runRec">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>●</string>
     </property>
    </widget>
    <widget class="Line" name="lineRecEyeTracking">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>290</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="chooseReplaySettings">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Felvétel megnyitása</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsViewUserName">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>350</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="userName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>215</y>
       <width>250</width>
       <height>30</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelUserName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Név:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="replayRec">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>⏵</string>
     </property>
    </widget>
    <zorder>graphicsViewRecEyeTracking</zorder>
    <zorder>runCalibration</zorder>
    <zorder>runRec</zorder>
    <zorder>chooseReplaySettings</zorder>
    <zorder>lineRecEyeTracking</zorder>
    <zorder>replayRec</zorder>
    <zorder>graphicsViewUserName</zorder>
    <zorder>labelUserName</zorder>
    <zorder>userName</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBoxRunningText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>740</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGraphicsView" name="graphicsViewRunningText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>740</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
    <widget class="Line" name="lineDown">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>79</y>
       <width>740</width>
       <height>2</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="lineRight">
     <property name="geometry">
      <rect>
       <x>739</x>
       <y>0</y>
       <width>2</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="lineUp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>740</width>
       <height>2</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="lineLeft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>2</width>
       <height>80</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="runningText">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>15</y>
       <width>940</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Futó szöveg...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <zorder>graphicsViewRunningText</zorder>
    <zorder>runningText</zorder>
    <zorder>lineDown</zorder>
    <zorder>lineRight</zorder>
    <zorder>lineUp</zorder>
    <zorder>lineLeft</zorder>
   </widget>
   <widget class="QLabel" name="circle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIsmertet_s">
    <property name="title">
     <string>Ismertetés</string>
    </property>
    <addaction name="actionhttps_github_com_galal1_repleye"/>
   </widget>
   <addaction name="menuIsmertet_s"/>
  </widget>
  <action name="actionSz_veg_hozz_ad_s">
   <property name="text">
    <string>Szöveg hozzáadása</string>
   </property>
  </action>
  <action name="actionSz_veg_ment_se">
   <property name="text">
    <string>Szöveg mentése</string>
   </property>
  </action>
  <action name="action_llj_Stop">
   <property name="text">
    <string>Állj / Stop</string>
   </property>
  </action>
  <action name="actionSebess_g">
   <property name="text">
    <string>Sebesség</string>
   </property>
  </action>
  <action name="actionSz_k_z_sz_less_ge">
   <property name="text">
    <string>Szóköz szélessége</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionSz_vegm_ret">
   <property name="text">
    <string>Szövegméret</string>
   </property>
  </action>
  <action name="actionBe_ll_t_sok_ment_se">
   <property name="text">
    <string>Beállítások mentése</string>
   </property>
  </action>
  <action name="actionBe_ll_t_sok_t_rl_se">
   <property name="text">
    <string>Beállítások törlése</string>
   </property>
  </action>
  <action name="actionKalibr_ci_ind_t_sa">
   <property name="text">
    <string>Kalibráció indítása</string>
   </property>
  </action>
  <action name="actionFelv_tel_ind_t_sa">
   <property name="text">
    <string>Felvétel indítása</string>
   </property>
  </action>
  <action name="actionVisszaj_tsz_s">
   <property name="text">
    <string>Felvétel megnyitása</string>
   </property>
  </action>
  <action name="actionTeker_s_a_felv_telben">
   <property name="text">
    <string>Tekerés a felvételben előre</string>
   </property>
  </action>
  <action name="actionTeker_s_a_felv_telben_h_tra">
   <property name="text">
    <string>Tekerés a felvételben hátra</string>
   </property>
  </action>
  <action name="actionhttps_github_com_galal1_repleye">
   <property name="text">
    <string>https://github.com/galal1/repleye</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
